### DISCUSSION TITLE
More folder levels

### DISCUSSION THREAD

**Discussion by albnet73** [tumcms/forge-bim360.project.setup.tool#5] [General]
It's not clear if it's possible to add additional columns on the CSV file with another level_4 of folder.
I looked the script very quickly but i didn't found a limitation. can you confirm please if i can add it?

Thanks a lot in advance

AG

**Comment by seb-esser** [comment]
Hi @albnet73 our tool should not restrict this. However, we've experienced some refresh issues when staging complex folder breakdown structures (also see #4 ). We will follow up on this but the I expect that we can resolve any issues with some `System.Pause()` statements in case BIM360 needs to process our operations before allowing us to create the next folder. 
@DeianStoi fyi
